closed/OpenJ9.gmk

```diff
@@ -475,7 +475,7 @@ endef # RunJPP
 IncludeIfUnsure := -includeIfUnsure -noWarnIncludeIf
 
 $(J9JCL_SOURCES_DONEFILE) : \
-		$(foreach dir, $(JppSourceDirs), $(call RecursiveWildcard,$(dir),*.java))
+		$(foreach dir, $(JppSourceDirs), $(call recur_wildcard,$(dir),*.java))
```

Comment:
I suggest a better change is to rename the macro `recur_wildcard` (on line 436) to `RecursiveWildcard` for consistency with other jdk versions (probably from which this was copied).

Reply:
Renamed `recur_wildcard` to `RecursiveWildcard`.